enhancements/1186-spiffe-integration/README.md

```diff
@@ -0,0 +1,54 @@
+# KEP-1186: Spiffe integration in NFD
```

Comment:
Please change CRD to CR in the shole KEP and specify what object is suppose to be signed.

Reply:
Done!